README.md

```diff
@@ -37,6 +37,31 @@ $ docker run --detach \
              thelounge/thelounge:latest
 ```
 
+### Executing commands in the container
+
+Due to the way root permissions are dropped in the container, it's highly recommended to pass the `--user node` argument to any
+commands you execute in the container via Docker to ensure that file permissions retain the correct owner, like so:
+
+```
+$ docker exec --user node -it <container_name> thelounge add MyUser
```

Comment:
Don't use <> or Docker hub will strip it as a html tag.

Reply:
even if it's inside a code blocky?

Reply:
edd88e975a8dcf155bebe94bed690c39e3d63483

Reply:
gyad dyamit dockerhub